### PR TITLE
Added addDistance Method

### PR DESCRIPTION
**Pull request by Sri-V**
Issue 11 Add Method for Distance.

### CODE REVIEW

**Comment by Sri-V** [comment]
Added the required method, however i doubt that constructor Distance(float, float) is working properly. Kindly review and let me know if anything needs to be modified.

File: src/main/java/segelzwerg/sporttooolbox/IUnits/Distance.java [hunk 1]
```diff
@@ -13,6 +13,14 @@ public Distance(float kilometer, float meter) {
 		this.meter = meter % 1000;
 	}
 
+
+    public Distance addDistance(Distance toAdd)
+    {
+        float kilometer= this.kilometer + toAdd.kilometer + (int) (this.meter+toAdd.meter) / 1000;
+        float meter= (this.meter+toAdd.meter) % 1000;
```
Comment (Segelzwerg) [inline src/main/java/segelzwerg/sporttooolbox/IUnits/Distance.java]: Can you refactor it, so there is no duplication with the constructor meaning an extracting a method from it and replacing it in this method and the constructor. Or we could also just add the values and do the overflow in the constructor.

File: src/test/java/segelzwerg/sporttooolbox/SpeedCalculatorTest.java [hunk 2]
```diff
@@ -20,4 +20,14 @@ public void thirty_kilometer_one_hour_test_speed() {
...
+    @Test
+    public void add_distance_test() {
+        Distance thirtyKilometer = new Distance(30,999);
+        Distance thirtyonekilometer= thirtyKilometer.addDistance(new Distance(0,1));
+
+        Distance expected=new Distance(31,0);
+
+        assertThat(thirtyonekilometer, equalTo(expected));
+    }
 }
```
Comment (Segelzwerg) [inline src/test/java/segelzwerg/sporttooolbox/SpeedCalculatorTest.java:33]: This should be in DistanceTest so we later can find test easier corresponding to the classes.
And you are almost right. The distance class was missing an annotation for creating the equals and hashCode method. This was my bad, but I fixed in 4a714a3

**Review by Segelzwerg — CHANGES_REQUESTED** [review]
Thank you for contributing. I left two comments on it to improve the code.

File: src/main/java/segelzwerg/sporttooolbox/IUnits/Distance.java (hunk 1 above)
Comment (Sri-V) [reply src/main/java/segelzwerg/sporttooolbox/IUnits/Distance.java]: done, for now i thought let constructor handle it.

File: src/test/java/segelzwerg/sporttooolbox/SpeedCalculatorTest.java (hunk 2 above)
Comment (Sri-V) [reply src/test/java/segelzwerg/sporttooolbox/SpeedCalculatorTest.java:33]: I saw a different issue raised for this so am not touching this part.

**Review by Segelzwerg — APPROVED** [review]

**Pull request merged**